index.js

```diff
@@ -425,4 +425,19 @@
     return JSON.parse(s);
   });
 
+  // Regex  //////////////////////////////////////////////////////////////////
+
+  //. match :: RegEx -> String -> Maybe [*]
+  //. Takes a regular expression and tests it against the supplied string
+  //. returning a Nothing if there is no match or a Just containing an
+  //. array of the the matched results.
+  //.
+  //. ```javascript
+  //. > S.match(/abcd/, 'abcdefg')
+  //. Just(['abcd'])
+  //. ```
+  S.match = R.curry(function(regex, s) {
+    return R.compose(S.toMaybe, R.match(regex))(s);
+  });
```

Comment:
Could this simply be defined as `S.match = R.compose(S.toMaybe, R.match);` (or the `R.pipe` equivalent)?